### DISCUSSION TITLE
New app icon feedback

### DISCUSSION THREAD

**Discussion by alexcmgit** [lakscastro/kanade#26] [Polls]
The v0.4.0 introduced a new icon.

This is a pool to collect some feedback, it may impact future releases.

## New icon

<img src="https://user-images.githubusercontent.com/51419598/204112344-956a6869-87cb-4cdc-a0c7-b9d956acb348.png" height="150" />

<div>

<img src="https://user-images.githubusercontent.com/51419598/204113447-9f41f38d-39a1-4c60-8859-b3956d17cf43.png" width="300" />

</div>

---

## Old icon

<img src="https://user-images.githubusercontent.com/51419598/204112408-aa52ac99-4711-4da5-b679-3930f8f82c27.png" height="150" />

<div>

<img src="https://user-images.githubusercontent.com/51419598/204113541-b815f1b4-2da9-4213-ab76-8547cacabf3f.png" width="300" />

</div>

**Comment by Thebear375** [comment]
![apk_extractor](https://user-images.githubusercontent.com/30216228/204118870-e1a464b5-e3a6-4a64-ba4c-25671706300e.png)
Something like this

**Reply by alexcmgit** [reply]
It looks like an really old app (from 2012, 2014), I also prefer to keep arrows and texts away from app icons. Thanks for the suggestion anyways.

**Comment by Wjxfi** [comment]
you just need to make the old icon round

Then it will be beautiful

as I tried to do here
![Polish_20221127_135136382.jpg](https://user-images.githubusercontent.com/113500363/204131565-1fc555f6-85d8-4678-b998-ca52450711ca.jpg)

**Comment by Wjxfi** [comment]
even the dumbest person would understand which icon is better
![288306.jpg](https://user-images.githubusercontent.com/113500363/204166154-d1706d58-63ae-408e-a452-d85ad4f1d4b2.jpg)
![288310.jpg](https://user-images.githubusercontent.com/113500363/204166159-ecabcdcf-d4ae-4405-b70a-6884b72f15b4.jpg)

please leave the previous one or round it up (miui automatically rounded it up)

**Comment by Wjxfi** [comment]
the new icon does not match the style of the application

**Comment by kotx** [comment]
It could just be me but the new icon appears blurry due to antialiasing in the launcher, also looks worse in monochrome.
![Screenshot_20221219-104038](https://user-images.githubusercontent.com/33439542/208496762-ed16de52-93ad-4f39-afdd-9a8f8c811328.png)

**Reply by kotx** [reply]
Also it might be worth it to have the icon version as a setting?

**Reply by alexcmgit** [reply]
@wstxda do you have any idea?

> Also it might be worth it to have the icon version as a setting?

Thanks but i don't think so, i don't like the idea of the same app with totally different icons. Different variants in last case.

**Reply by WSTxda** [reply]
> It could just be me but the new icon appears blurry due to antialiasing in the launcher, also looks worse in monochrome. ![Screenshot_20221219-104038](https://user-images.githubusercontent.com/33439542/208496762-ed16de52-93ad-4f39-afdd-9a8f8c811328.png)

Icons in laucher is based in you screen resolution, for it have mdpi, hdpi, xx, xxx variants in app source. and, icon in screen so small, why you need high resolution? :P

**Reply by WSTxda** [reply]
but, you can ask dev to drop kitkat support and migrate to vector drawable icons

**Reply by WSTxda** [reply]
> @WSTxda do you have any idea?
> 
> > Also it might be worth it to have the icon version as a setting?
> 
> Thanks but i don't think so, i don't like the idea of the same app with totally different icons. Different variants in last case.

nah, its not good just use one or other icons and keep visual identity
